cmd/ops_agent_uap_plugin/plugin.go

```diff
@@ -48,14 +79,18 @@ func main() {
 	server := grpc.NewServer()
 	defer server.GracefulStop()
 
-	ps := &OpsAgentPluginServer{server: server}
+	ps := &OpsAgentPluginServer{server: server, runCommand: runCommand}
 	// Successfully registering the server and starting to listen on the address
 	// offered mean Guest Agent was successful in installing/launching the plugin
 	// & will manage the lifecycle (start, stop, or revision change) here onwards.
 	pb.RegisterGuestAgentPluginServer(server, ps)
+
+	ctx := context.Background()
+	ps.GetStatus(ctx, &pb.GetStatusRequest{})
```

Comment:
Do we actually need to call the GetStatus and Start over here?
I understood that it was the plugin manager task to issue those commands. We can also leave like this and change later if need bee.